packages/mini-browser/src/node/mini-browser-endpoint.ts

```diff
@@ -186,6 +196,14 @@ export class MiniBrowserEndpoint implements BackendApplicationContribution, Mini
         };
     }
 
+    protected getVirtualHostRegExp(): RegExp {
```

Comment:
Can we return `Promise<RegExp>`?
In Che, it requires one more place to check for a virtual host - through the async call.
It may be useful for extenders in other downstreams as well.